pages/resultsPage.py

```diff
@@ -0,0 +1,59 @@
+from datetime import date,timedelta
+
+class connection():
+    def __init__(self, stations,date,deparr,duration,transfers):
+        self.stations=stations
+        self.date=date
+        self.deparr=deparr
+        self.duration=duration
+        self.transfers=int(transfers)
```

Comment:
Choć Python jest językiem dynamicznie typowanym, to warto trzymać typy w ryzach i zawsze deklarować jakich spodziewamy się wartości przy atrybutach:
```suggestion
        self.transfers: int =int(transfers)
```
czy  parametrach metod (jak i zwracanych przez nie wartościach):
```
createConnectionObject(self, row: int) -> WebElement:
```